PasswordValidator/Validators/AdvancedValidator.cs

```diff
@@ -1,10 +1,17 @@
 ﻿using PasswordValidator.Enums;
+using PasswordValidator.Rules;
 
 namespace PasswordValidator.Validators
 {
     public class AdvancedValidator : IPasswordValidator
     {
         public ValidatorType ValidatorType => ValidatorType.Advanced;
+        public IEnumerable<IRule> Rules { get; set; }
```

Comment:
private readonly - means it can only be created in the constructor and not assigned to again